components/filter/filter.js

```diff
@@ -322,6 +336,14 @@ class Filter extends FocusMixin(LocalizeCoreElement(RtlMixin(LitElement))) {
 
 		const dimension = this._getActiveDimension();
 
+		const introductoryTextClasses = {
+			'd2l-body-compact': true,
+			'd2l-filter-dimension-intro-text': true,
+			'd2l-filter-dimension-intro-text-multi-dimension': !singleDimension
```

Comment:
As discussed in the team channel, sounds like we could change this to `multi-dimension` and the one below to `with-intro` and get things a little shorter and cleaner 🎉